tests/select_for_update/tests.py

```diff
@@ -179,6 +179,20 @@ def test_for_update_requires_transaction_only_in_execution(self):
         with self.assertRaises(transaction.TransactionManagementError):
             list(people)
 
+    @skipUnlessDBFeature('supports_select_for_update_with_limit')
+    def test_select_for_update_with_limit(self):
+        other = Person.objects.create(name='Other')
```

Comment:
"Reinhardt" (the name of the first user here) is the last name of Django (the guitarist). Use `name="Grappeli"` (Stephane, who used to play with Reinhardt).